Library/Homebrew/cask/audit.rb

```diff
@@ -488,7 +488,17 @@ def audit_signing
 
           next unless path.exist?
 
-          result = system_command("spctl", args: ["--assess", "--type", "install", path], print_stderr: false)
+          result = case artifact
+          when Artifact::Pkg
+            system_command("spctl", args: ["--assess", "--type", "install", path], print_stderr: false)
+          when Artifact::App
+            system_command("spctl", args: ["--assess", "--type", "execute", path], print_stderr: false)
+          when Artifact::Binary
+            system_command("codesign",  args:         ["-vvvv", "-R=notarized", "--check-notarization", path],
```

Comment:
I think checking for notarisation should be a separate (strict) check. My understanding is it's not required for ARM/Gatekeeper/Quarantine.

The message below can also probably remove "notarize" from it, too.